assets/src/components/book_table.js

```diff
@@ -24,6 +26,10 @@ class BookTable extends Component {
   }
 }
 
+const BookTable = ({ data: { loading, books }, ...rest }) => (
+  <BaseBookTable loading={ loading } books={ books } { ...rest } />
+)
```

Comment:
I think we could trim off some mental overhead by renaming components like these that only have to do with data fetching, so:

`BookTable` -> `BookQuery` 
`NestedBookTable` -> `NestedBookQuery`

Furthermore, since the two components are identical save the `nested` prop, if we could find some way to get the prop in, we could wrap the same component. If we keep the data prop the same we could just wrap the base table:

``` javascript
export const BookQuery = graphql(topLevelQuery)(BaseBookTable)
export const NestedBookQuery = graphql(nestedQuery)(withProps({ nested: true })(BaseBookTable))
```

This might be too fancy though.

Reply:
I like the idea (especially about using that `withProps`, I wasn't aware of that one before). 

But, I think I'm held up on the naming -- `BookQuery` and `NestedBookQuery` make sense here, but when specifying the hierarchy in `app.js`, it gets a little awkward to have `hierarchy={[BookQuery, NestedRegionQuery, SecurityQuery]}` (maybe that means the specifying of the hierarchy is the issue, though).

Perhaps we can circle back and refactor this a bit later?